web/ui/formatted-text.svelte

```diff
@@ -20,5 +20,179 @@
   .formatted-text :global(img) {
     max-width: 100%;
     height: auto;
+    padding: 0.3em 0;
+    margin: 0 auto;
+  }
+
+  .formatted-text :global(p) {
+    margin: 0 0 0.857em;
+  }
+
+  .formatted-text :global(h1) {
+    margin: 1em 0;
+    font-size: 1.476em;
+  }
+
+  .formatted-text :global(h2) {
+    margin: 1em 0;
+    font-size: 1.429em;
+  }
+
+  .formatted-text :global(h3) {
+    margin: 1em 0;
+    font-size: 1.296em;
+  }
+
+  .formatted-text :global(h4) {
+    margin: 1em 0;
+    font-size: 1.215em;
+  }
+
+  .formatted-text :global(h5) {
+    margin: 1em 0;
+    font-size: 1.138em;
+  }
+
+  .formatted-text :global(h6) {
+    margin: 1em 0;
+    font-size: 1.067em;
+  }
+
+  .formatted-text :global(a) {
+    color: var(--link-color);
+    text-decoration: none;
+    word-wrap: break-word;
+    opacity: 100%;
+    transition:
```

Comment:
Having `color` transition on `:hover` makes UI slow. since UI has no instant reaction of user’s actions.

Animations are cool, but we should use them to explain UI changes.